uber/templates/preregistration/confirm.html

```diff
@@ -54,13 +54,10 @@
         </div>
       {% endif %}
 
-      {# Deprecated form included for backwards compatibility with old plugins #}
```

Comment:
This is just moved to be underneath the other fields. Doing that here (and in the other pages) fixes Javascript used in regextra.html in our plugins, which was written assuming regextra.html was at the end of all the other fields.